packages/ui/src/services/color-config/color-theme-presets.ts

```diff
@@ -25,18 +28,21 @@ export const colorsPresets: Record<string, EssentialVariables> = {
     // Accent
     primary: '#00b4d8',
     secondary: '#767C88',
-    success: '#40e583',
-    info: '#2c82e0',
-    danger: '#de1041',
-    warning: '#ffac0a',
-    // Text Colors
-    textDark: '#242424',
-    textLight: '#f1f1f1',
+    success: '#3D9209',
+    info: '#158DE3',
+    danger: '#E42222',
+    warning: '#FFD43A',
+
     // Background Colors
     backgroundPrimary: '#1F1F1F',
-    backgroundSecondary: '#333333',
-    backgroundTertiary: '#262626',
-    backgroundElement: '#767C88',
+    backgroundSecondary: '#262626',
+    backgroundElement: '#262626',
+    backgroundBorder: '#363636',
+
+    // Text Colors
+    textPrimary: '#f1f1f1',
```

Comment:
```suggestion
    textPrimary: '#F1F1F1',
```

Reply:
I think stylelint must handle it, not me.